### DISCUSSION TITLE
Wezterm rendering of Github's Monaspace font with adjusted line/cell height

### DISCUSSION THREAD

**Discussion by johnend** [wezterm/wezterm#5791] [Q&A] [answered]
### What Operating System(s) are you running on?

macOS

### WezTerm version

20240203-110809-5046fc22

### Ask your question!

So for some reason when using the GitHub Monaspace font's I get some inconsistent rendering for some characters.

When I say inconsistent in this instance I mean different on different lines. Not necessarily consistently inconsistent per character.

For example:
The M in the Colemak (line 3)
![image](https://github.com/wez/wezterm/assets/3886057/4c7c5b27-3a58-4789-8c9b-be75e10deaa8)

And after scrolling to a different line:
![image](https://github.com/wez/wezterm/assets/3886057/d84e804b-ed68-4326-9556-6ca7791980c6)

Another example with the T in adjust:
![image](https://github.com/wez/wezterm/assets/3886057/fe082611-e4b5-4873-9a4d-f09e45bd8eeb)
![image](https://github.com/wez/wezterm/assets/3886057/d54ff350-abf5-4b7f-850c-00fbff1f7777)

You can see in all of the above screenshots that there are other examples of letters rendering differently here too. This can also occur with S and P characters.

I have searched to see if I could find other folks with the same issue but came up empty, and I didn't want to raise an issue because I'm not certain whether this is a Wezterm thing, the fonts themselves, or a NeoVim thing (haven't _noticed_ any issues outside of NeoVim, but that might just be because that's where I live 😂 ). My assumption is that is a font issue rather than a Wezterm one as otherwise there would be issues with other fonts.

Anyway I would prefer to have a slightly larger cell height to improve readability on my monitor, but this bugs me to the extent that I'd rather not adjust it. Any help would be greatly appreciated. 😄

**Comment by wez** [comment, marked as answer]
This sort of thing is usually related to hinting. See https://wezfurlong.org/wezterm/config/lua/config/freetype_load_flags.html?h=hinting

**Reply by johnend** [reply]
Hmm, OK I did try a few of those options when initially setting up, and none render the offending letters at the right heights, so my assumption is that it must be an issue with the font itself. Thanks for the response! I have added a comment on an issue in their repo.
